### DISCUSSION TITLE
Pinecil v2 - Handle temperature reading higher than v1

### DISCUSSION THREAD

**Discussion by dogtopus** [Ralim/IronOS#1805] [Q&A]
So I recently got a Pinecil v2 and the first thing I noticed is that the handle temperature seems to be off. With Pinecil v1 the cold temperature (the "I plug it in the first time today and it pops up" temperature reading) is normally around my room temperature and it's fairly accurate, while the Pinecil v2 is significantly higher (around 8 degC higher than room and v1). This is seen on both the 2.21 and RC firmware. I also took it apart and inspected the joints and they seem to be fine.

Is this a known firmware issue or is there something wrong with my hardware?

**Comment by dogtopus** [comment]
Measuring the NTC, the voltage divider and their connection to the microcontroller (directly to the pins) with a multimeter also ended pretty uneventfully. Everything seems to be within spec.

I wonder if [this](https://github.com/Ralim/IronOS/pull/1529) has something to do with it, in a sense that the calibration (coe) for the ADC is not the same as the one @Ralim used to calculate the NTC LUT.

**Reply by Ralim** [reply]
I'm highly suspect that PR does indeed correlate to the issue. Thank you for cross checking with a multimeter to the 3380 tables.

I've not yet dug into updating that PR and doing cleanup passes on it sadly though :/
